CHANGELOG.asciidoc

```diff
@@ -20,6 +20,7 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha2...master[Check the HEAD d
 *Packetbeat*
 
 *Topbeat*
+- Export CPU stats per core in type=core documents and add `cpu_usage_in_ticks` option {pull}1705[1705]
```

Comment:
I suggest to split this up into two lines, as these are 2 changes (even though it is in on PR).